src/defs/metrics.d.ts

```diff
@@ -42,6 +42,20 @@ declare module "tsdDef" {
     }
 
     /**
+     * Interface for classes which create <code>Metrics</code> instances. Clients
```

Comment:
Does nodejs support similar html doc generation as java? Or is this just copy-paste? I'm fine continuing the annotations Mohammed has adopted, however, we should remove the html mark-up unless it actually gets used.